### ISSUE TITLE
FairRoot with light installation of FairSoft (w/o simulation engines)

### ISSUE THREAD

**Issue by vfriese** [FairRootGroup/FairRoot#907]
FairSoft offers the possibility to be installed without simulation engines (Geant3 / Geant4); this can be choosen in the questionnaire conducted by configure.sh. However, FairRoot does not compile against such a "light" FairSoft (see below), which makes the feature somehow useless.

Are there plans to help this situation by making FairRoot compilable w/o simulation engines?

Here is the compilation error with FairRoot v18.2.0 against FairSoft jun19p1 (light):
```
[ 62%] Linking CXX shared library ../../lib/libFairMCStepLogger.dylib
[ 62%] Built target FairMCStepLogger
[ 62%] Generating G__FairFastSimDict.cxx, G__FairFastSimDict_rdict.pcm, ../../../lib/libFairFastSim.rootmap
input_line_12:19:10: fatal error: 'TG4VUserFastSimulation.h' file not found
#include "TG4VUserFastSimulation.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
Error: /Users/vfriese/Software/fairsoft/jun19p1_nosim/bin/rootcint: compilation failure (/Users/vfriese/Software/fairroot/build/base/sim/fastsim/G__FairFastSimDictd69d857cf3_dictUmbrella.h)
make[2]: *** [base/sim/fastsim/G__FairFastSimDict.cxx] Error 1
make[1]: *** [base/sim/fastsim/CMakeFiles/FairFastSim.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by dennisklein** [comment]
Thx for your bug report!

**Comment by dennisklein** [comment]
We believe the issue has been fixed in the `v18.2.1` release and the `dev` branch.
